ReadMe.md

```diff
@@ -55,69 +55,69 @@ For CLI usage please see [Manual.md](Manual.md).
 
 ## Default Config
 
-Config file location by default `/etc/clickhouse-backup/config.yml` can be redefined via `CLICKHOUSE_BACKUP_CONFIG` environment variable
-All options can be overwritten via environment variables
-Use `clickhouse-backup print-config` for print current config
+By default, the config file is located at `/etc/clickhouse-backup/config.yml`, but it can be redefined via `CLICKHOUSE_BACKUP_CONFIG` environment variable.
+All options can be overwritten via environment variables.
+Use `clickhouse-backup print-config` to print current config.
 
 ```yaml
 general:
   remote_storage: none           # REMOTE_STORAGE, if `none` then `upload` and  `download` command will fail
-  max_file_size: 1073741824      # MAX_FILE_SIZE, 1G by default, useless when upload_by_part is true, use for split data parts files by archives  
-  disable_progress_bar: true     # DISABLE_PROGRESS_BAR, show progress bar during upload and download, have sense only when `upload_concurrency` and `download_concurrency` equal 1
-  backups_to_keep_local: 0       # BACKUPS_TO_KEEP_LOCAL, how much newest local backup should keep, 0 mean all created backups will keep on local disk
-                                 # you shall to run `clickhouse-backup delete local <backup_name>` command to avoid useless disk space allocations
-  backups_to_keep_remote: 0      # BACKUPS_TO_KEEP_REMOTE, how much newest backup should keep on remote storage, 0 mean all uploaded backups will keep on remote storage. 
-                                 # if old backup is required for newer incremental backup, then it will don't delete. Be careful with long incremental backup sequences.
-  log_level: info                # LOG_LEVEL, allow `debug`, `info`, `warn`, `error`
+  max_file_size: 1073741824      # MAX_FILE_SIZE, 1G by default, useless when upload_by_part is true, use to split data into archives of defined size
```

Comment:
`Data parts` here is important, do you know about system.parts table?

Reply:
Not sure which line is it for?